cypress/support/step_definition/tasks/createEditAndDeleteTasks.spec.js

```diff
@@ -2,21 +2,13 @@ import Tasks from '../../pages/tasks/pageTasks'
 
 const tasks = new Tasks()
 
-//Scenario: Teacher creates task as draft from room
-//Given ('I am logged in as a 'teacher1' at 'brb'')
-//step defined -->\step_definition\authentication\loginStep.spec.js
-
-//When I go to rooms overview
-//step defined -->\step_definition\course\commonCourseSteps.spec.js
-
-//And I go to room 'Course with subject and tasks'
-//step defined -->\step_definition\course\commonCourseSteps.spec.js
-
-//And('I click on FAB to create new content')
-//step defined -->\step_definition\course\commonCourseSteps.spec.js
-
-//And('I click on New Task')
-//step defined -->\step_definition\course\commonCourseSteps.spec.js
+// EXTERNAL STEP DEFINITIONS
```

Comment:
here the title can be-> EXTERNAL COMMON STEP DEFINITIONS

Reply:
done